installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned.adoc

```diff
@@ -55,8 +55,6 @@ include::modules/cluster-telemetry.adoc[leveloffset=+1]
 
 * See xref:../../../support/remote_health_monitoring/about-remote-health-monitoring.adoc#about-remote-health-monitoring[About remote health monitoring] for more information about the Telemetry service
 
```

Comment:
I just wanted to check, but did you intend to totally remove the external load balancer information from this assembly?

Reply:
Hi @EricPonvelle . Yes, myself and @bscott-rh did not see the value of the modules in this assembly as it's high level and not a dedicated install route.